Products/CMFPlone/browser/contact_info.py

```diff
@@ -67,12 +67,13 @@ def send_message(self, data):
 
         try:
             # This actually sends out the mail
-            host.send(
+            host.secureSend(
```

Comment:
Ah, right. I guess alternatively we could set `message['Reply-to']` for the same effect.